examples/nlp/language_modeling/conf/megatron_gpt_prompt_learning_config.yaml

```diff
@@ -36,6 +37,13 @@ exp_manager:
     filename: 'megatron_gpt_prompt_tune--{val_loss:.3f}-{step}'
     model_parallel_size: ${model.tensor_model_parallel_size}
     save_best_model: True
+  early_stopping_callback_params:
```

Comment:
can we move this out of `exp_manager` OR move the callback code inside exp_manager?

Reply:
(discussed offline) will update exp_manager core in a separate PR.